### ISSUE TITLE
External Temperature Sensor

### ISSUE THREAD

**Issue by Jonnehs** [speleolontra/daikin_residential_altherma#190]
Hello,

My Daikin has a separate wired temperature sensor installed to help the unit measure the true outside temperature, this is required/recommended when the unit is in direct sunlight as the sensor on the unit can be pretty inaccurate once the unit casing is heating up in sunlight.

It shows on my indoor controller unit, snip picture below, can it be pulled from the API into your integration do you know? It would be handy to have as data inside HASS.

Thanks!

![image](https://github.com/speleolontra/daikin_residential_altherma/assets/11739441/8abd518f-0502-4ce2-b196-e0c804f80d56)

**Comment by jwillemsen** [comment]
Sorry, but that temperature is not exported by daikin to the cloud so we can't make it visible in HA, see the device diagnostics, please report a feature request to Daikin to show it in the onecta app, when it is visible there we should be able to make it visible within HA.

Be aware that the external outdoor temperature is only used for the heating curve, for deciding weather the tank will be heated using the outdoor unit or the backup heater *always* the sensor within the outdoor unit will be used, not your external one.

**Comment by MicheleMercuri** [comment]
if you have installed a wired external sensor you will see (in Onecta app and in HA) only temperature measured by this one and not temperature measured by onboard sensor on external unit as you can see below: 
![1](https://github.com/speleolontra/daikin_residential_altherma/assets/97225788/e985c029-6b74-4e11-b7ea-b4d7d056d61f)
![2](https://github.com/speleolontra/daikin_residential_altherma/assets/97225788/cfe8e7b5-5f8a-4eba-b89a-e57455cada96)
![1](https://github.com/speleolontra/daikin_residential_altherma/assets/97225788/3089cdd4-f1d2-40eb-a022-8b6ec6e37440)

**Comment by jwillemsen** [comment]
Yes, you are right, but you will not see the other outside temperature anymore
